components/general/Hero.tsx

```diff
@@ -13,7 +13,10 @@ const Hero = () => {
             digital agency
             <br /> from Melbourne.
           </div>
-          <div className={`${textFont.className} global__text-xlarge ${styles["hero__txt"]}`}>
+          <div
+            className={`${textFont.className} global__text-xlarge ${styles["hero__txt"]}`}
+            aria-label="hero__txt"
```

Comment:
Why did you change this hero component? Is it related to your card?

Reply:
It caused testing failed when others add a new section with the same text, so I changed it. I will remove my changes.

Reply:
If it is a problem unrelated to your card, consider creating a new bug ticket. Otherwise, change it if it is truly necessary (not recommended).